tts/tts-adapters/tts-adapter-sapinative/src/main/java/org/daisy/pipeline/tts/sapi/impl/SAPIService.java

```diff
@@ -82,16 +78,27 @@ protected void activate() {
 
 	@Override
 	public TTSEngine newEngine(Map<String,String> properties) throws Throwable {
+		if (SAPI_SAMPLERATE == null)
+			SAPI_SAMPLERATE = Properties.getProperty("org.daisy.pipeline.tts.sapi.samplerate",
+					true,
+					"Audio sample rate of legacy Windows voices (in Hz)",
+					false,
+					"22050");
+		if (SAPI_BYTESPERSAMPLE == null)
+			SAPI_BYTESPERSAMPLE = Properties.getProperty("org.daisy.pipeline.tts.sapi.bytespersample",
+					true,
+					"Audio bit depth of legacy Windows voices (in bytes per sample)",
+					false,
+					"2");
+		if (SAPI_PRIORITY == null)
+			SAPI_PRIORITY = Properties.getProperty("org.daisy.pipeline.tts.sapi.priority",
+					true,
+					"Priority of Windows voices relative to voices of other engines",
+					false,
+					"7");
```

Comment:
These variables have are initialized in the `activate()` method. I guess you added it because there was a problem running the tests? Could you call `activate()` in the tests instead?

Reply:
Oh yes indeed it was because of that.
Will change that asap then

Reply:
@bert Done !

(I also removed a test of synthesis with Onecore native adapter that I forgot to remove)